tutorials/R/Geospatial-skills/primer-raster-r/Introduction-to-Raster-Data-In-R/Introduction-to-Raster-Data-In-R.Rmd

```diff
@@ -398,8 +406,12 @@ used to quickly define a crop extent.
 
 #plot the DEM
 plot(DEM)
+
 #Define the extent of the crop by clicking on the plot
-cropbox1 <- drawExtent()
+cropbox1 <- drawExtent(show=TRUE, col="red")
```

Comment:
Okay, I'm going to guess here that you were having trouble with the function, and you copied the `drawExtent(show=TRUE, col="red")` from the ?drawExtent() help file, or from another example somewhere? That's what I would do. However, in the ?drawExtent() function documentation, such as [shown here](https://www.rdocumentation.org/packages/raster/versions/3.0-12/topics/drawExtent) the function and arguments shown are actually _the default arguments_ that do not need to be added at all (i.e., if you do not add any arguments to `drawExtent()` it uses the options (arguments) of show=TRUE and col="red" by default, so in this line of code they are actually redundant (An can be removed, please, reverting this line to its original form). The convention for the help files is that the data input values are shown as argument names without anything assigned to them, and the 'options' arguments are shown with their default assignments simply to be helpful and informative to the user who is trying to learn about that function. For example, [`the lm()` function](https://www.rdocumentation.org/packages/stats/versions/3.6.2/topics/lm) (which calculates a linear model for a dataset, such as a best fit line through as scatterplot) has many required (`formula`) or optional (`weights, subset, data`, etc.) data inputs, and several arguments that are shown with their defaults (`method = "qr", model = TRUE, x = FALSE, y = FALSE, qr = TRUE`, etc.). Hope that's helpful!